ui/shared/TransactionPreview.qml

```diff
@@ -25,6 +25,7 @@ Item {
     property bool isGasEditable: false
     property bool isValid: fromValid && gasValid
```

Comment:
**Actually** `toValid` has to be in this expression as well. However, that will bring us back to the issue that, if `toValid` is false (which is what we want in case we do a chat command for send transaction), the modal will be impossible to be submitted..